docs/cluster-management/configure-queue-service.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: main
+title: Configuring the Queue Service
+category: Cluster Management
+menu: menu
+toc: 
+    - title: Managing the Queue Service
+      url: "#managing-the-queue-service"
+      active: true
+    - title: Packages
+      url: "#packages"
+    - title: Configuration
+      url: "#configuration"
+---
+# Managing the Queue Service
+
+## Packages
+
+Like the other services, the API is shipped as a [Docker image](https://hub.docker.com/r/screwdrivercd/queue-service/) with port 80 exposed.
+
+```bash
+$ docker run -d -p 7000:80 screwdrivercd/queue-service:latest
+$ open http://localhost:7000
```

Comment:
I think this should be `9003` which is consistent to 

```
 httpd:
  port: 9003
  host: 0.0.0.0
  uri: http://YOUR_IP:9003
```

from the `queue-service` config

Reply:
> I think this should be `9003` which is consistent to
> 
> ```
>  httpd:
>   port: 9003
>   host: 0.0.0.0
>   uri: http://YOUR_IP:9003
> ```
> 
> from the `queue-service` config

Updated to 80 there, in PR.